### DISCUSSION TITLE
Can't verify myself as student

### DISCUSSION THREAD

**Discussion by niklasbrecht** [community/community#84209] [GitHub Education]
### Hi everyone,

Hello there,
Your form does not allow me to verify myself with my student ID. It has my name on the front and proof of the current semester on the back. As your guidelines only allow me to upload a single photo (!), my application is automatically rejected. How can I verify myself now? There was no answer in the Community Forum.
I cannot upload a photo with front and back, as edited photos are rejected.

**Comment by fury-05** [comment]
You can ask your college to issue a study right certificate. OR

Try this it may work : 

You can try taking a photo copy of your student ID on same page (Front and back) and then you can upload it as single pic.

**Comment by InvisiblePro** [comment]
Any proof of your institution can be accepted by GitHub but requires a date i.e. 10-Oct-20xx mentioned on the proof. It will take at least 1 week after you submit your documents/proofs. Even though I didn't have my school ID, I have been approved as a Student because it requires proof with a date. And also need to update your [GitHub billing address](https://github.com/settings/billing/payment_information).
Have a good journey ✨
If this worked, please make sure to "mark the answer".
